governator-test-spock/src/main/java/com/netflix/governator/test/GovernatorExtension.java

```diff
@@ -0,0 +1,68 @@
+package com.netflix.governator.test;
+
+import org.spockframework.runtime.extension.AbstractGlobalExtension;
+import org.spockframework.runtime.model.SpecInfo;
+
+import com.netflix.governator.guice.test.AnnotationBasedTestInjectorManager;
+import com.netflix.governator.guice.test.InjectorCreationMode;
+import com.netflix.governator.guice.test.mocks.MockHandler;
+import com.netflix.governator.test.mock.spock.SpockMockHandler;
+
+import spock.lang.Specification;
+
+public class GovernatorExtension extends AbstractGlobalExtension {
```

Comment:
Yeah, going to run through now and add javadoc and readmes. But this is a type for a ServiceLoader to find which binds our framework to the test class.